libsatadrivers/include/satadrivers/virtio/virtio_blk.h

```diff
@@ -0,0 +1,126 @@
+/* This header is BSD licensed so anyone can use the definitions to implement
```

Comment:
consider adding an SPDI tag

Reply:
Yes, please. Here the tag should be added, not replacing the existing header (only replace for our own headers)

Reply:
Yep, also have to fix other files as well.